element_array_ephys/probe.py

```diff
@@ -122,6 +141,14 @@ def build_electrodes(site_count, col_spacing, row_spacing,
 
 @schema
 class Probe(dj.Lookup):
+    """Represent a physical probe with unique ID
+
+    Attributes:
+        probe (foreign key, varchar(32) ): Unique ID for this model of the probe.
+        ProbeType (query): ProbeType entry.
```

Comment:
Can this be a `dict` key? Or does it need to be a query object? If changed, may require changes elsewhere
```suggestion
        ProbeType (dict): ProbeType entry.
```

Reply:
Yeah I wasn't sure about this. I think I can use a dict. I think I've gone back and forth so I'll make sure everything is consistent